pkgs/applications/misc/feedbackd/default.nix

```diff
@@ -1,46 +1,63 @@
 { lib
 , stdenv
 , fetchFromGitLab
+, docbook-xsl-nons
+, gobject-introspection
+, gtk-doc
+, libxslt
 , meson
 , ninja
 , pkg-config
+, vala
 , wrapGAppsHook
 , glib
 , gsound
-, libgudev
 , json-glib
-, vala
-, gobject-introspection
+, libgudev
+, dbus
 }:
 
 stdenv.mkDerivation rec {
-  pname = "feedbackd-unstable";
-  version = "2021-01-25";
+  pname = "feedbackd";
+  version = "0.0.0+git20210426";
```

Comment:
I know other distros do this, but it breaks nixpkgs rules.
from the manual's [package naming secition](https://nixos.org/manual/nixpkgs/stable/#sec-package-naming):
```
 If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23". 
```
I'm still learning about nixpkgs conventions, however, so maybe its okay.

Reply:
In this case this is a release – well, maybe not a project release but it is an official package release.

Reply:
Ohh I see, so in this case we follow upstreams version scheme

Reply:
Well, technically, those are downstream releases (packages from Debian) but since they are maintained by upstream in the upstream repo and tags are created for them, it is easier for us to just use them.

Though, maybe this fact would deserve a comment.

Reply:
What kind of comment do you have in mind?

Reply:
Maybe something like: “Not an actual upstream release, only a Debian package release that is tagged in the upstream repo.”